internal/tools/aws/database_multitenant.go

```diff
@@ -206,36 +208,36 @@ func (d *RDSMultitenantDatabase) Provision(store model.InstallationDatabaseStore
 
 	err = d.createDatabaseIfNotExist(ctx, installationDatabaseName)
 	if err != nil {
-		return errors.Wrapf(err, "unable to create schema in multitenant RDS cluster ID %s", *lockedRDSCluster.cluster.DBClusterIdentifier)
+		return errors.Wrap(err, "failed to provision multitenant database")
 	}
 
 	installationSecret, err := d.ensureMultitenantDatabaseSecretIsCreated(lockedRDSCluster.cluster.DBClusterIdentifier, vpc.VpcId)
 	if err != nil {
-		return errors.Wrapf(err, "failed to get a secret for installation ID %s", d.installationID)
+		return errors.Wrap(err, "failed to provision multitenant database")
 	}
 
 	err = d.createUserIfNotExist(ctx, installationSecret.MasterUsername, installationSecret.MasterPassword)
 	if err != nil {
-		return errors.Wrapf(err, "failed to create a Mattermost database schema for installation ID %s", d.installationID)
+		return errors.Wrap(err, "failed to provision multitenant database")
 	}
 
 	err = d.grantUserFullPermissions(ctx, installationDatabaseName, installationSecret.MasterUsername)
 	if err != nil {
-		return errors.Wrapf(err, "failed to grant permissions to Mattermost database user for installation ID %s", d.installationID)
+		return errors.Wrap(err, "failed to provision multitenant database")
 	}
 
 	databaseInstallationIDs, err := store.AddMultitenantDatabaseInstallationID(*lockedRDSCluster.cluster.DBClusterIdentifier, d.installationID)
 	if err != nil {
-		return errors.Wrapf(err, "failed to add installation ID %s to the multitenant database ID %s", d.installationID, *lockedRDSCluster.cluster.DBClusterIdentifier)
+		return errors.Wrap(err, "failed to provision multitenant database")
 	}
 
 	err = d.updateCounterTag(lockedRDSCluster.cluster.DBClusterArn, len(databaseInstallationIDs))
 	if err != nil {
-		return errors.Wrapf(err, "failed to update tag:counter in RDS cluster ID %s", *lockedRDSCluster.cluster.DBClusterIdentifier)
+		return errors.Wrap(err, "failed to provision multitenant database")
```

Comment:
Why be less descriptive with all these errors? Won't using `failed to provision multitenant database` in so many places make it difficult for us to find which exact line logged the error when debugging?

Reply:
Because we already have the IDs information in the logging fields. Unless I misunderstood @gabrieljackson [here](https://github.com/mattermost/mattermost-cloud/pull/226#pullrequestreview-420574030), I thought that the goal was to reduce the size of these messages by using fields.

Reply:
I agree with shorter errors and using the logging fields as much as possible but we should do that without losing details on the error. Also for all these error returns, they're not going to get the logger fields added in this function since we're not logging, just returning an error

Reply:
We are logging when they bubble up to the provisioner.

Reply:
Right, I know that but now there's 5-10 log messages that all look the same and it will be hard to track where they originated

Reply:
Right, I don't disagree with having unique as possible error messages but I think we should have a discussion on how standardized the error/logs.